tests/test_sourmash.py

```diff
@@ -199,7 +252,6 @@ def test_do_compare_quiet(c):
     testdata1 = utils.get_test_data('short.fa')
     testdata2 = utils.get_test_data('short2.fa')
     c.run_sourmash('compute', '-k', '31', testdata1, testdata2)
-
```

Comment:
note - please try to avoid changes in bits of the code that aren't relevant to the issues tackled by this PR - it adds a bit of confusion to the review :)

Reply:
Sorry, it's probably something I did before and forgot to revert! Should I fix it and push again?

Reply:
no need, just for future consideration!